lib/board.rb

```diff
@@ -15,4 +15,96 @@ def create_cells
             end
         end
     end
-end
\ No newline at end of file
+
+
+
+  
+#     def initialize
+#       @cells = create_cells
+#     end
+  
+#     def create_cells
+#       cells = {}
+#       all_coordinates.each do |coordinate|
+#         cells[coordinate] = Cell.new(coordinate)
+#       end
+#       cells
+#     end
+
+  
+#     def all_coordinates
+#       coordinates = []
+#       rows.each do |row|
+#         columns.each do |column|
+#           coordinates << "#{row}#{column}"
+#         end
+#       end
+#       coordinates
+#     end
+  
+
+#     def rows
+#       ["A", "B", "C", "D"]
+#     end
+  
+
+#     def columns
+#       [1, 2, 3, 4]
+#     end
+#   end
```

Comment:
This is a cool alternative way to create the hash. We can change it if you prefer this version.